libraries/ssh_crypto.rb

```diff
@@ -153,7 +154,7 @@ def valid_macs # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLen
     when 'redhat', 'centos', 'oracle'
       case inspec.os[:release]
       when /^6\./
-        macs = macs53
+        macs = macs53_el65
```

Comment:
Lets add a comment to reference the redhat announcement here and document that we consider 6.x prior to 6.5 as outdated and not maintained